engine/gamesys/src/gamesys/components/comp_model.cpp

```diff
@@ -356,7 +356,7 @@ namespace dmGameSystem
                 texture_res = GetTextureFromSamplerNameHash(&component->m_Resource->m_Materials[material_index], material, i, material->m_SamplerNames[i]);
             }
 
-            ro->m_Textures[i] = texture_res->m_Texture;
+            ro->m_Textures[i] = texture_res ? texture_res->m_Texture : 0;
```

Comment:
This code was altered in e5d97f32ad984dd996193a4dca28e7503c44ebda
Restoring it to the previous state.